js/web/lib/wasm/jsep/webgpu/ops/conv-grouped.ts

```diff
@@ -84,13 +100,14 @@ export const createGroupedConvProgramInfo =
   }`;
       return {
         name: 'GroupedConv',
-        shaderCache: {hint: attributes.cacheKey},
+        shaderCache: {hint: `${isChannelLast}`, inputDependencies},
```

Comment:
Similar to conv2d, what about the activation/clipMax/clipMin?

Reply:
Done

Reply:
The `activation/clipMax/clipMin` is still missing for conv-grouped.